src/main/fc/fc_msp.c

```diff
@@ -1338,7 +1340,24 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         }
 #endif
         break;
-
+#ifdef USE_RCSPLIT
+    case MSP_RCSPLIT_BOXNAMES:
+        serializeRCSplitBoxNamesReply(dst);
+        break;
+    case MSP_RCSPLIT_BOXIDS:
+        serializeRCSplitBoxIdsReply(dst);
+        break;
+    case MSP_RCSPLIT_MODE_RANGES:
+        for (int i = 0; i < MAX_RC_SPLIT_MODE_ACTIVATION_CONDITION_COUNT; i++) {
+            const modeActivationCondition_t *mac = rcSplitModeActivationConditions(i);
+            const box_t *box = findRCSplitBoxByBoxId(mac->modeId);
+            sbufWriteU8(dst, box->permanentId);
+            sbufWriteU8(dst, mac->auxChannelIndex);
+            sbufWriteU8(dst, mac->range.startStep);
+            sbufWriteU8(dst, mac->range.endStep);
+        }
+        break;
+#endif
```

Comment:
Using current RC_MODE (after extending available bits) will work with current 'Modes' tab. Standard configurator will handle that, no need for change